quellen-sol/solden-bot
# Finalize repo (#1)

* Add rest of jobs

* Add eslint/prettier

* Run lint

* Just use format script on commit

* Add CircleCI

* Script fix

* Test bump script

* Passing check

* Add error handling

* Add error handling to bot event listener

* Admin user id
